app/views/admin/tracks/index.html.haml

```diff
@@ -15,6 +15,9 @@
         %th Color
```

Comment:
I'd move this column closer to the title, so that the first columns are either the same or similar to the original view, ie
name, descr, color, room, dates, submitter, state, actions
I don't think we need to show short_name at all.